python/cudf/cudf/core/series.py

```diff
@@ -5216,3 +5218,47 @@ def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
         result_col[equal_nulls] = True
 
     return Series(result_col, index=index)
+
+
+def factorize(values, sort=False, na_sentinel=-1):
```

Comment:
Can we add the `size_hint` parameter just to be completely compatible with Pandas?

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.factorize.html

Reply:
This also still feels like a very awkward place for this to live. Any ideas on an alternative?

Reply:
Done